ci/watcher_v1beta1_watcher.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: watcher.openstack.org/v1beta1
+kind: Watcher
+metadata:
+  name: watcher
+spec:
+  databaseInstance: "openstack"
+  tls:
```

Comment:
after #66 merged, the tls field is part of `apiServiceTemplate`

Reply:
Thanks!